docs/guides/documentation/releases/README.md

```diff
@@ -0,0 +1,94 @@
+# Releases
+
+<pre align="center">Streamline software releases through consistent templates, automated aggregation and notifications.</pre>
+
+## Introduction
+
+**Background**: Making releases can be a chore. From ensuring the release notes have helpful information to aggregating many updates and notifying your users - doing the process right can be time consuming and fraught with misunderstandings. In this guide we recommend a standardized, automated way to notify your users of new releases. For example, we recommend using a `.github/release.yml` template to allow for more readable release notes, showcasing the importance of each update. We also provide automation for aggregating releases from many repositories into a super-release that provides a holistic view of project progress. Finally, we recommend automated notifications through GitHub integrations to help keep everyone informed.
+
+**Use Cases**:
+- Automating release notes generation with a readable template
+- Aggregating multiple repository releases for projects with many repositories
+- Streamlining release notifications to keep teams and users informed automatically
+
+---
+
+## Prerequisites
+* A GitHub account and repository
```

Comment:
Suddenly changed from `-` bullets to `*` bullets? The marked up rendering is the same, sure, but Markdown is supposed to be readable as plain text too. Maybe make these consistent? 🤔

Reply:
Good catch - and again this is actually from [our SLIM Guide Template](https://nasa-ammos.github.io/slim/docs/contribute/submit-best-practice#use-our-standard-guide-template). Open to improvements based on your suggestions!

Reply:
To be resolved at the template level: https://github.com/NASA-AMMOS/slim/issues/161